examples/nlp/intent_slot_classification/conf/intent_slot_classification_config.yaml

```diff
@@ -0,0 +1,91 @@
+# Sentence classification with pretrained BERT models
+
+trainer:
+  gpus: 1 # the number of gpus, 0 for CPU
+  num_nodes: 1
+  max_epochs: 100
+  max_steps: null # precedence over max_epochs
+  accumulate_grad_batches: 1 # accumulates grads every k batches
+  amp_level: O0 # O1/O2 for mixed precision
+  distributed_backend: ddp
+  checkpoint_callback: False  # Provided by exp_manager
+  logger: False  # Provided by exp_manager
+
+model:
+  data_dir: /home/vgetselevich/data/intent_slots/atis/nemo-processed # /path/to/data
+  class_balancing: null # or weighted_loss
+  intent_loss_weight: 0.6 # only if class_balancing = weighted_loss
```

Comment:
It is not related to weighted loss and should work always. You may drop the comment.

Reply:
Resolved

Reply:
Added